src/Instrumentation/HttpAsyncClient/_register.php

```diff
@@ -2,4 +2,17 @@
 
 declare(strict_types=1);
 
-\OpenTelemetry\Contrib\Instrumentation\HttpAsyncClient\HttpAsyncClientInstrumentation::register();
+use OpenTelemetry\Contrib\Instrumentation\HttpAsyncClient\HttpAsyncClientInstrumentation;
+use OpenTelemetry\SDK\Sdk;
+
+if (class_exists(Sdk::class) && Sdk::isInstrumentationDisabled(HttpAsyncClientInstrumentation::NAME) === true) {
+    return;
+}
+
+if (extension_loaded('otel_instrumentation') === false) {
```

Comment:
@brettmc opened PR that changes name of extension, after https://github.com/open-telemetry/opentelemetry-php-instrumentation/pull/49 it will be `opentelemetry`, we have to synchronize between the two.

Reply:
I would say, lets merge this first as the other PR still needs to get merged and a new version tag has to be made available. Renaming is a good choice imho, but it requires A LOT more changes through the repo which could be an easy search-replace

Reply:
maybe it's better to add extension name as constant variable to `open-telemetry/api` - it's required in all packages?